src/main/java/software/amazon/encryption/s3/S3EncryptionClient.java

```diff
@@ -379,7 +400,13 @@ public DeleteObjectsResponse deleteObjects(DeleteObjectsRequest deleteObjectsReq
      */
     @Override
     public CreateMultipartUploadResponse createMultipartUpload(CreateMultipartUploadRequest request) {
-        return _multipartPipeline.createMultipartUpload(request);
+        try {
+            return _multipartPipeline.createMultipartUpload(request);
+        } catch (CompletionException e) {
+            throw new S3EncryptionClientException(e.getCause().getMessage(), e.getCause());
+        } catch (Exception e) {
+            throw new S3EncryptionClientException("Unable to delete objects.", e);
```

Comment:
I guess the error message here is wrong for this method `createMultipartUpload`. Instead, we should be throwing `e.getMessage()`.